mobile/src/screens/welcome.js

```diff
@@ -0,0 +1,180 @@
+'use strict'
+
+import React, { Component } from 'react'
+import { Image, Linking, StyleSheet, Text, View } from 'react-native'
+import { connect } from 'react-redux'
+import { fbt } from 'fbt-runtime'
+import SafeAreaView from 'react-native-safe-area-view'
+import RNSamsungBKS from 'react-native-samsung-bks'
+
+import { createAccount } from 'actions/Wallet'
+import Disclaimer from 'components/disclaimer'
+import OriginButton from 'components/origin-button'
+import CommonStyles from 'styles/common'
+
+const IMAGES_PATH = '../../assets/images/'
+
+class WelcomeScreen extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false
+    }
+  }
+
+  handleCreateWallet = async () => {
+    this.setState({ loading: true }, () => {
+      setTimeout(() => {
+        this.props.createAccount()
+        this.setState({ loading: false })
+        this.props.navigation.navigate('Authentication')
+      })
+    })
+  }
+
+  handleImportWallet = async () => {
+    this.setState({ loading: false })
+    this.props.navigation.navigate('ImportAccount')
+  }
+
+  render() {
+    let action
+    if (this.props.samsungBKS.seedHash === '') {
+      action = this.renderSamsungBKSRequiresSetup()
+    } else if (
+      this.props.samsungBKS.seedHash &&
+      this.props.samsungBKS.seedHash.length > 0
+    ) {
+      action = this.renderSamsungBKSDetectedMessage()
+    } else if (this.props.wallet.accounts.length === 0) {
+      action = this.renderWalletButtons()
+    } else {
+      action = this.renderContinueButton()
+    }
+
+    return (
+      <SafeAreaView style={styles.content}>
+        <View style={{ ...styles.container, flexGrow: 2 }}>
+          <Image
+            resizeMethod={'scale'}
+            resizeMode={'contain'}
+            source={require(IMAGES_PATH + 'origin-dark-logo.png')}
+            style={styles.image}
+          />
+          <Text style={styles.title}>
```

Comment:
I think this one's supposed to be `#0b1823`. 😐

Reply:
Is that the colour of the title?

Reply:
Yeah, but apparently not the titles at the top of other screens 🤷‍♂️ I like the darker one anywhere.